docs/WifiPerformance.md

```
@@ -60,71 +60,135 @@ All wireless adapters were tested under consistent conditions - each positioned
+    [  5]   0.00-10.02  sec   204 MBytes   170 Mbits/sec                  receiver
+    
+    iperf Done.
+    ```
+=== "Wireless link info"
+    ```
+    freq: 2437
+    RX: 105104 bytes (432 packets)
+    TX: 50924 bytes (228 packets)
+    signal: -34 dBm
+    rx bitrate: 300.0 MBit/s MCS 15 40MHz short GI
+    
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 4

(MD009, no-trailing-spaces)